apache-httpclient/pom.xml

```diff
@@ -63,6 +63,11 @@
             <version>${wiremock.version}</version>
             <scope>test</scope>
         </dependency>
```

Comment:
So, remove this

Reply:
Do you refer to remove this dependency?
`<dependency>
            <groupId>org.apache.httpcomponents</groupId>
            <artifactId>httpclient</artifactId>
            <version>${httpclient.version}</version>
        </dependency>`

Reply:
Yes - But you can leave it